src/log_levels.h

```diff
@@ -132,6 +132,7 @@ enum log_sev_t
    LOUTPUT   = 101, //! output
    LUNC      = 102, //! rules used in uncrustify.cpp
    LQT       = 103, //! load/save options for Qt
+   LMINFLOOP = 104, //! mod_infinite_loop
```

Comment:
I think we don't need a specific log option for this, unless you can explain a good use case for it. Using 'log_rule_B' should suffice, but happy to hear @guy-maurel 's opinion as well.

Reply:
I have removed the LOG_FMT calls as requested.